### PR TITLE
Bug 1270442 - Update browser tab title. r=yifan

### PR DESCRIPTION
**Pull request by danhuang1202**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3e6b69380b89e506e90fe4f3f3b1060ef98c143b

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4df1821ca61b9302f59f5f6bacace1a9f740884e

File: tv_apps/browser/locales/browser.en-US.properties [hunk 1]
```diff
@@ -39,7 +39,7 @@ start-browsing=Enter a search term or Web address to start browsing.
 # LOCALIZATION NOTE: depending on your locale, you might want to use a more
 # compact translation for top-sites-tab than for top-sites-startPage.
 top-sites-startPage = Top sites
-top-sites-tab       = Top sites
+top-sites-tab       = Top Sites
```
Comment (begeeben) [inline tv_apps/browser/locales/browser.en-US.properties]: Should use a new l10n ID.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=125388d80411366c2764dec144fbd78a6571faa8

File: tv_apps/browser/locales/browser.en-US.properties (hunk 1 above)
Comment (danhuang1202) [inline tv_apps/browser/locales/browser.en-US.properties]: Updated

**Pull request merged**